src/js/components/settings/organization/organization.js

```diff
@@ -48,6 +48,25 @@ export const OrgHeader = () => {
   );
 };
 
+export const CopyTextToClipboard = ({ token }) => {
+  const [copied, setCopied] = useState(false);
+  const { classes } = useStyles();
+
+  const onCopied = () => {
+    setCopied(true);
+    setTimeout(() => setCopied(false), 5000);
```

Comment:
nit pick:magic number, could we have a const for that?

Reply:
https://github.com/mendersoftware/gui/pull/3211/commits/4f079cb7dc04d6b31fa8d0c8c3bac7ae0a2eca04 <- should address this for... several locations... 🙈